### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by JTY009**
Updated to version of file-upload

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -336,6 +356,286 @@ Request (POST)
...
+```
+
+| Responses    | Status    | Description |
+| --------|---------|-------|
+| Ok  | 200   | Successfully update event.
+| Bad Request  | 400   | Invalid Event.
+| Locked  | 423   | Cannot update event. Routing request is already received for this envelope.
+
+#### EXAMPLE (FOR FILE IN QUARANTINE)
+Request (POST): localhost:8898/file-upload/events/FileQuarantinedStored
```
Comment (hannesstockner) [inline README.md]: I think FileQuarantinedStored does not exist.

File: README.md [hunk 2]
```diff
@@ -336,6 +356,286 @@ Request (POST)
...
+``` json
+{
+    "callbackUrl": "string representing absolute url",
+    "metadata": { "any": "valid json object" }
+}
+```
+
+Response (in Headers): Location → localhost:8898/file-upload/envelopes/0b215e97-11d4-4006-91db-c067e74fc653
+
+#### MANUALLY CHANGE STATE
```
Comment (hannesstockner) [inline README.md]: MANUALLY SEND COMMANDS

File: README.md [hunk 3]
```diff
@@ -336,6 +356,286 @@ Request (POST)
...
+
+| Responses    | Status    | Description |
+| --------|---------|-------|
+| Ok  | 200   | Successfully changed a "SEALED" envelope to "OPEN".
+| Bad Request | 400   | Incorrect Command.
+| Locked | 423   | Envelope already routed and received.
+
+
+#### EXAMPLE
+Request (POST): localhost:8898/file-upload/unsealed
```
Comment (hannesstockner) [inline README.md]: Command name is unsealenvelope

File: README.md [hunk 4]
```diff
@@ -336,6 +356,286 @@ Request (POST)
...
+{
+    "callbackUrl": "string representing absolute url",
+    "metadata": { "any": "valid json object" }
+}
+```
+
+Response (in Headers): Location → localhost:8898/file-upload/envelopes/0b215e97-11d4-4006-91db-c067e74fc653
+
+#### MANUALLY CHANGE STATE
+Currently, this endpoint changes the state of only "SEALED" envelopes to "OPEN" to allow resubmissions of files to that envelope.
```
Comment (hannesstockner) [inline README.md]: I think we should remove this text

File: README.md (hunk 1 above)
Comment (JTY009) [inline README.md]: corrected

File: README.md (hunk 2 above)
Comment (JTY009) [inline README.md]: changed

File: README.md [hunk 5]
```diff
@@ -336,6 +356,285 @@ Request (POST)
...
+| --------|---------|-------|
+| Ok  | 200   | Successfully return list of envelopes
+
+#### EXAMPLE
+Request (GET): localhost:8898/file-upload/envelopes
+
+Response: 200
+
+#### CREATE ENVELOPE WITH ID
+Creates a substitute envelope with the original envelope's Id for files to upload to. This will only create an envelope if the original endpoint failed to create an envelope.
```
Comment (hannesstockner) [inline README.md]: lets remove the description

File: README.md [hunk 6]
```diff
@@ -21,23 +23,26 @@ The endpoints can then be accessed with the base url http://localhost:8898/
 *   [Endpoints](#endpoints)
 *   [Callback](#callback)
 *   [Envelope Event Statuses](#events)
+*   [Test-Only Endpoints](#testonly)
+*   [Internal Automated Endpoint](#auto)
```
Comment (hannesstockner) [inline README.md]: Internal Inter Communication Endpoint

File: README.md [hunk 7]
```diff
@@ -154,6 +155,23 @@ Response (in Body):
 }
 ```
 
+#### Upload File (DO NOT USE)
```
Comment (hannesstockner) [inline README.md]: should be under the inter communication section

File: README.md [hunk 8]
```diff
@@ -336,6 +339,302 @@ Request (POST)
...
+
+#### MANUALLY SEND COMMANDS
+
+```
+POST    /file-upload/commands/{commandType}
+```
+
+| Responses    | Status    | Description |
+| --------|---------|-------|
+| Ok  | 200   | Successfully changed a "SEALED" envelope to "OPEN".
```
Comment (hannesstockner) [inline README.md]: Lets change description to: Command successfully processed

**Pull request merged**